certification/internal/engine/openshift.go

```diff
@@ -183,7 +189,7 @@ func (pe OpenshiftEngine) CreateCatalogSource(data cli.CatalogSourceData, opts c
 	}, nil
 }
 
-func (pe OpenshiftEngine) DeleteCatalogSource(name string, opts cli.OpenshiftOptions) (*cli.OpenshiftReport, error) {
+func (pe openshiftEngine) DeleteCatalogSource(name string, opts cli.OpenshiftOptions) (*cli.OpenshiftReport, error) {
 	crdClient, err := client.CatalogSourceClient(pe.KubeConfig, opts.Namespace)
```

Comment:
NIT: `crdcClient` seems like a copy-paste ism, and probably should be something else.